core/DataAccess/LogAggregator.php

```diff
@@ -1075,6 +1087,17 @@ public function queryConversionsByDimension($dimensions = array(), $where = fals
         $orderBy = false;
         $query   = $this->generateQuery($select, $from, $where, $groupBy, $orderBy);
 
+        if (!empty($rankingQuery)) {
+            $sumColumns = array_keys($availableMetrics);
+            $rankingQuery->addColumn($sumColumns, 'sum');
```

Comment:
@diosmosis can we assume that it is always a `sum`? or is it maybe also sometimes min, max, etc?

Reply:
missed this :+1: